### DISCUSSION TITLE
My SSL installation works for Google Chrome but not for Edge and Firefox.

### DISCUSSION THREAD

**Discussion by eguvenc** [FiloSottile/mkcert#570] [Q&A]
I performed the windows installation as follows.

https://scoop.sh/

```
Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
Invoke-RestMethod -Uri https://get.scoop.sh | Invoke-Expression
```

```sh
mkcert -install
mkcert example.local '*.example.local' localhost 127.0.0.1 ::1
```

I work on VmWare apache2 and I installed mkcert on windows.

```sh
SSLEngine on
SSLCertificateFile  /etc/ssl/localsite.test.pem
SSLCertificateKeyFile /etc/ssl/localsite.test-key.pem
```
After the installation, It works for Google Chrome.
But;
Not works for Microsoft Edge.
Not works for Firefox.

I tried to install manually or other browsers , but i could not find the .CA file.

**Comment by wnunezc** [comment]
Well, in my very particular case, each developer has to deploy their development environment, this means that each one goes through the process of installing their personal certificate to be able to use it in the development environment.

I solved your problem by creating the certificate on the machine I want to access, and then copying those certificates to the web service servers and configuring them to be implemented as ssl.
